charts/platform/values.yaml

```diff
@@ -45,20 +45,22 @@ hostAliases: []
   #   hostnames:
   #     - service.local.lab
 
-# -- The pod security context
+# -- The pod security context (https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod)
 podSecurityContext:
```

Comment:
If you wanted to offer more secure defaults, we put these on the pod level

```
      securityContext:
        runAsNonRoot: true
        seccompProfile:
          type: RuntimeDefault
```

Reply:
Updated to add those.